lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php

```diff
@@ -36,26 +36,30 @@
     /**
      * @var SimpleXMLElement
```

Comment:
Thank you for your feedback! Actually it still should be a `SimpleXMLElement`, but I completely understand this is a bit unclear. This is actually something I improved in my last commits, without knowing you were reviewing an older version. I do think it should answer your interrogation (you might want to check the [GenerateSitemapXML method](https://github.com/byng-systems/pimcore-sitemap-plugin/pull/6/commits/96c8cfe49a7a271e59dc5cfbcaf821a3e00fa275#diff-71d0e7681cf36c448120f554e8c70f94R97) for example). 

Let me know if you think it still needs work!